examples/src/main/scala/org/http4s/blaze/examples/NIO2HttpServer.scala

```diff
@@ -12,10 +12,10 @@ import scala.concurrent.duration._
 
 class NIO2HttpServer(port: Int) {
 
-  val group = AsynchronousChannelGroup.withFixedThreadPool(10, java.util.concurrent.Executors.defaultThreadFactory())
+  val group = AsynchronousChannelGroup.withFixedThreadPool(6, java.util.concurrent.Executors.defaultThreadFactory())
```

Comment:
Would using some multiple of the available processors from `Runtime` be appropriate here?